website/docs/guides/versioned-resources.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "kubernetes"
+page_title: "Versioned resource names"
+description: |-
+  This guide explains the naming conventions for resources and data sources in the Kubernetes provider. 
+---
+
+# Versioned resource names 
+
+This guide explains the naming conventions for resources and data sources in the Kubernetes provider. 
+
+
+## Version suffixes
+
+From provider version v2.7.0 onwards Terraform resources and data sources that cover the [core set of Kubernetes APIs](https://kubernetes.io/docs/reference/kubernetes-api/) will be suffixed with their corresponding Kubernetes API version (e.g `v1`, `v2`, `v2beta1`). The existing resources in the provider will continue to be maintained as is. 
```

Comment:
Kubernetes also has an API group called "core".

In this case, I would use "standard" or some other term to avoid confusion with the actual API group and not imply that it only applies to those resources.